src/Toggle/Entity/ActivatableToggle.php

```diff
@@ -1,7 +1,7 @@
 <?php
 namespace Clearbooks\Labs\Toggle\Entity;
 
-interface ActivatableToggle
+interface ActivatableToggle extends Toggle
```

Comment:
Seems sensible to me.. this is an entity object.

All ```ActivatableToggle```s are ```Toggle```s but not all ```Toggle```s are activatable.